imports/plugins/core/payments/server/no-meteor/resolvers/Query/availablePaymentMethods.js

```diff
@@ -0,0 +1,10 @@
+import { decodeShopOpaqueId } from "@reactioncommerce/reaction-graphql-xforms/shop";
+import { paymentMethods } from "/imports/plugins/core/core/server/no-meteor/pluginRegistration";
+
+export default async function (_, { shopId }, context) {
+  const allPaymentMethods = Array.from(paymentMethods);
+  const dbShopId = decodeShopOpaqueId(shopId);
+  const shop = await context.queries.shopById(context, dbShopId);
+  return shop.availablePaymentMethods
+    .map((availableName) => allPaymentMethods.find(({ name }) => name === availableName));
```

Comment:
All of these lines except the `decodeShopOpaqueId` line should be moved to a new `availablePaymentMethods` query function in this plugin. We want to keep all real logic out of the resolvers. This function should only be:

```js
const dbShopId = decodeShopOpaqueId(shopId);
return context.queries.availablePaymentMethods(context, dbShopId);
```

See https://docs.reactioncommerce.com/docs/next/graphql-create-query